components/SatisfactionSlider.tsx

```diff
@@ -41,35 +41,27 @@ const SatisfactionSlider = ({
 
   return (
     <>
-      <fieldset className={styles.wrapper} disabled={disabled}>
-        {title && (
-          <legend>
-            <h2>
-              <span className={styles.invisibleA11yLabel}>
-                {t('Slider.valueDescription')}
-              </span>{' '}
-              {title}
-            </h2>
-          </legend>
-        )}
-        <Smiley percentage={scaledNumber} />
-        <input
-          aria-hidden="true"
-          name={title}
-          type="hidden"
-          value={scaledNumber}
-        />
-        <input
-          aria-label={title}
-          max={max}
-          min={min}
-          name={title}
-          onChange={(e) => updateNumber(parseInt(e.target.value))}
-          step={step}
-          type="range"
-          value={number}
-        />
-      </fieldset>
+      <span className={styles.invisibleA11yLabel}>
+        {t('Slider.valueDescription')}
+      </span>{' '}
+      <Smiley percentage={scaledNumber} />
+      <input
+        aria-hidden="true"
+        disabled={disabled}
+        name={title}
+        type="hidden"
+        value={scaledNumber}
+      />
```

Comment:
Why is this here? If it is hidden and disabled

Reply:
Good catch, this was moving the `disabled` from the `fieldset` to the `input`! Just moved it to the wrong one whoops.